ideas/76-smooth-ui.md

```diff
@@ -0,0 +1,81 @@
+Status app should be fast enough to provide user smooth and nice experience - no lags and UI freezes.
+
+
+## Swarm Participants
+<!-- Each contributor pledges to the idea with their FOCUS value. (hours per week) -->
+<!-- Here all roles in swarm are defined and filled, one of the contributors should responsibility of the Idea as Lead. -->
+
+<!-- Testing/Evaluation support role is also mandatory to check in on specified Goal dates or earlier. -->
+
+<!-- Lead Contributor is the Owner of the Idea. If required, they can get support from a PM, but should be responsible for end to end execution of the Idea. This includes ensuring appropriate resources are allocated, setting realistic timelines and milestones, and any post-launch metrics or bug fixes that are attributed to the Idea -->
+<!-- A swarm requires at minimum 3 contributors and 1 evaluator/tester -->
+<!-- 'Contributor' should be replaced with a descriptive role type. -->
+- Lead Contributor: @dmitryn
+- Testing & Evaluation: @Serhy
+- Contributor: @rasom 
+- Contributor: @janherich 
+- Contributor: @yenda
+- Contributor: @mandrigin 
+- Contributor: @flexsurfer 
+- UX: tba
+<!-- - Contributor: @username -->
+
+## Product Overview
+<!-- A short (~200 word) description and motivation of the Idea. Without clear explanation the Idea should not proceed. Can include User Stories -->
+<!-- Testing/Evaluation role accepts responsbility to checkin at Goal dates, -->
+<!-- forces discussion to continue implementation or recommend disband and post-mortem. -->
+App should be fast from user standpoint - switching screens, sending/receiving messages, dapp browsing. Every UI component should be responsible and provide almost immediate feedback to the user. For long-running operations spinner should be shown.
+
+User stories to be resolved:
+- https://github.com/status-im/status-react/issues/3095
+- https://github.com/status-im/status-react/issues/3108
+- https://github.com/status-im/status-react/issues/3397
+
+<!-- ### Product Description -->
+<!-- What functionality are you adding? What will this look like from a user perspective? Why is this important? -->
+
+### Requirements & Dependencies
+<!-- Are there bugs or feature requests in other repositories that are part of this Idea? -->
+<!-- There is no approval unless the idea requires to be reviewed by supporting organelles (Financial, Hiring, or Design). -->
+<!-- The Swarm must develop a fully fleshed out Requirements document for the idea to proceed, to the satisfaction of participants. -->
+Somewhat depends on 87-new-protocol idea to be finished to validate perfomance user story about getting many messages from offline (app freezes while doing that) https://github.com/status-im/status-react/issues/3095
+
+### Minimum Viable Product
+<!-- Mandatory, completes the Idea in the fastest route possible, can be hacky, needed to feel progress. See https://imgur.com/a/HVlw3 -->
+Goal Date: 2018-05-01
```

Comment:
Can we decompose this into something that can be delivered by Apr 9 (next town hall) and April 16? May 1 is a long time away. What does some partial step towards solving those 3 perf stories look like?